### PR TITLE
No lists and links in trail scribe editor

### PR DESCRIPTION
**Pull request by Reettaphant**
- Only bold and italics should be allowed for scribe trail text editor
- Copy to trail button should be removed because it's not being used much and stripping lists and links out on copy is too much work for functionality which does not get use
- I noticed that the `textArea` component was still here, I've removed it because we are now using the scribe editor instead.

### CODE REVIEW

File: public/video-ui/src/components/FormComponents/CopyTrailButton.js [hunk 1]
```diff
@@ -1,17 +0,0 @@
-import React from 'react';
```
Comment (akash1810) [inline public/video-ui/src/components/FormComponents/CopyTrailButton.js]: 👌

File: public/video-ui/src/components/FormFields/ScribeEditor.js [hunk 2]
```diff
@@ -184,29 +158,37 @@ export class ScribeEditor extends React.Component {
   }
 
   configureScribe() {
+
+    const allKeyboardShortcuts = {
+      bold: function(event) {
+        return event.metaKey && event.keyCode === 66;
```
Comment (akash1810) [inline public/video-ui/src/components/FormFields/ScribeEditor.js]: could we add these [here](https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/constants/keyCodes.js)?

Comment (Reettaphant) [inline public/video-ui/src/components/FormFields/ScribeEditor.js]: Yes!

File: public/video-ui/src/components/FormFields/ScribeEditor.js [hunk 3]
```diff
@@ -174,39 +144,38 @@ export class ScribeEditor extends React.Component {
...
-      this.refs.editor.innerHTML = nextProps.copiedValue;
-      this.setState({
-        copiedValue: nextProps.copiedValue
-      });
+  configureScribe() {
+
+    const allKeyboardShortcuts = {
+      bold: function(event) {
+        return event.metaKey && event.keyCode === keyCodes.b
+      }, // b
```
Comment (akash1810) [inline public/video-ui/src/components/FormFields/ScribeEditor.js]: probably don't need these comments anymore...

**Pull request merged**